### DISCUSSION TITLE
Purchase and Family Sharing

### DISCUSSION THREAD

**Discussion by arialdomartini** [MobiusSync/MobiusSync#189] [Q&A]
I purchased using the in-app option. I later discovered this implies the purchase is not family-shared. As far as I could read, that was not mention in the documentation.

I found no way to ask for a refund so I can purchase again using the Apple App Store. 

What would you suggest to do?

**Comment by SimonPickup** [comment]
In-app purchase are not family-shareable by default.

Apple makes it quite onerous to support family sharing.

Furthermore, only Apple can offer you a refund. See this link, or equivalent in your country:

https://support.apple.com/en-us/118223

**Comment by arialdomartini** [comment]
Thank you! Got the refund and immediately re-purchased.

**Comment by SimonPickup** [comment]
@arialdomartini How did repurchasing help?

**Reply by justin8** [reply]
I believe the möbius sync pro app does support family sharing, but the in-app purchase via the free one does not. I’m waiting for my refund now

**Comment by SimonPickup** [comment]
Thanks @justin8, yes Family Sharing is supported in the (prepaid) Pro version, but not the in-app purchase version.

It's annoying that Apple doesn't make it easier to do this.

**Comment by SimonPickup** [comment]
Raised an enhancement request: #208
